site/content/in-dev/unreleased/quickstart.md

```diff
@@ -97,42 +97,52 @@ To start using Polaris in Docker, launch Polaris while Docker is running:
 
 ```shell
 cd ~/polaris
-docker compose -f docker-compose.yml up --build
+./gradlew clean :polaris-quarkus-server:assemble -Dquarkus.container-image.build=true
+docker run -p 8181:8181 -p 8182:8182 apache/polaris:latest
 ```
 
-Once the `polaris-polaris` container is up, you can continue to [Defining a Catalog](#defining-a-catalog).
+You should see output for some time as Polaris builds and starts up. Eventually, you won’t see any more logs and should see messages that resemble the following:
+
+```
+INFO  [io.quarkus] [,] [,,,] (Quarkus Main Thread) Apache Polaris Server <version> on JVM  (powered by Quarkus <version>) started in 2.656s. Listening on: http://localhost:8181. Management interface listening on http://0.0.0.0:8182.
+INFO  [io.quarkus] [,] [,,,] (Quarkus Main Thread) Profile prod activated.
+INFO  [io.quarkus] [,] [,,,] (Quarkus Main Thread) Installed features: [...]
+```
+
+Once the container is up, you can continue to [Defining a Catalog](#defining-a-catalog).
 
 ### Building Polaris
 
-Run Polaris locally with:
+Run Polaris locally in [Quarkus Dev mode](https://quarkus.io/guides/dev-mode-differences) with:
 
 ```shell
 cd ~/polaris
-./gradlew runApp
+./gradlew --console=plain :polaris-quarkus-service:quarkusDev
```

Comment:
yikes. Does the invocation really need to be so lengthy?

Reply:
Yes.

Reply:
I mean, you can always create a shell alias or a run configuration in your IDE.

Reply:
Why can't a different gradle task just be used?

Reply:
Alright, I created 2 task aliases : `polarisServerRun` and `polarisServerDev`. Does that work for you?